src/domains/collections.ts

```diff
@@ -0,0 +1,290 @@
+import { gql, request } from "graphql-request";
+import { Extents, AssetLinks, STACCollection } from "../@types/StacCollection";
+import {
+  Collection,
+  CollectionsInput,
+  FacetGroup,
+} from "../models/GraphQLModels";
+import { mergeMaybe, buildClientId, scrubTokens } from "../utils";
+import { cmrSpatialToExtent } from "./bounding-box";
+
+const CMR_ROOT = process.env.CMR_URL;
+const GRAPHQL_URL = process.env.GRAPHQL_URL ?? "http://localhost:3003/api";
```

Comment:
By default current graphql endpoint for local dev is on http://localhost:3013/api